core/src/main/java/io/questdb/cairo/sql/Record.java

```diff
@@ -396,17 +396,6 @@ default long getUpdateRowId() {
         throw new UnsupportedOperationException();
     }
 
-    /**
-     * Reads bytes from varchar-specific storage and prints them into UTF8 encoded
-     * sink.
-     *
-     * @param col      numeric index of the column, 0-based
-     * @param utf8Sink the destination sink
-     */
-    default void getVarchar(int col, Utf8Sink utf8Sink) {
```

Comment:
the only beneficiary of this call was HTTP CSV export implementation, which can simply use getVarcharA() instead (and it does that).

However, because this method was on the interface, every varchar function had to implement it. Which was unfair. This was also tricky to test, due to the only real beneficiary.

The trigger for this was a function impl I could not figure out how to test.

Does this make sense?